packages/shared/tailwind/colors.js

```diff
@@ -20,6 +20,7 @@ const colors = {
     70: '#05B5C5',
     80: '#02AABD',
     90: '#009FB3',
+    lightLabel: 'var(--theme-label-primary)'
```

Comment:
@idoshamun Tagged you to see if this approach is ok, since colors could vary per dark/light color this made sense.

Reply:
Can you give an example of where we're going to use this?

Reply:
Its in this PR a specific color button on lightmode has different color text.

Reply:
Is it the one on `/tailwind/button.js`?

Reply:
I just saw the Slack conversation. This looks good. Thank you! 🚢